configure

```diff
@@ -2996,7 +3010,49 @@ main()
 
 		enable_sandbox_01=0
 	fi
+	
+	# Check the method used for returning complex numbers
+    if [ "x${complex_return}" = "xdefault" ]; then
+        if [ -n "${FC}" ]; then
+            # Determine the complex return type from the given Fortran compiler
+
+            # Query the full vendor version string output. This includes the
+            # version number along with (potentially) a bunch of other textual
+            # clutter.
+            # NOTE: This maybe should use merged stdout/stderr rather than only
+            # stdout. But it works for now.
+            vendor_string="$(${FC} --version 2>/dev/null)"
+        
+            # Query the compiler "vendor" (ie: the compiler's simple name) and
+            # isolate the version number.
+            # The last part ({ read first rest ; echo $first ; }) is a workaround
+            # to OS X's egrep only returning the first match.
+            fc_vendor=$(echo "${vendor_string}" | egrep -o 'ifort|GNU' | { read first rest ; echo $first ; })
+            
+            if [ "x${fc_vendor}" = "xifort" ]; then
+                complex_return='intel'
+            elif [ "x${fc_vendor}" = "xGNU" ]; then
+                complex_return='gnu'
+            else
+                echo "${script_name}: unable to determine Fortran compiler vendor!"
+                complex_return='gnu'
+            fi
+        else
+            complex_return='gnu'
+        fi
+    fi
+	
+    if [ "x${complex_return}" = "xgnu"  ]; then
+        complex_return_intel01='0'
+    elif [ "x${complex_return}" = "xintel" ]; then
+        complex_return_intel01='1'
+	else
+        echo "${script_name}: unknown complex return type \"${complex_return}\"! Cannot continue."
```

Comment:
Can we make this default to `gnu` if not given to not break scripts building BLIS?

Reply:
@isuruf it does default to `gnu` if `FC` isn't specified explicitly. Do you mean to use `gnu` even if `FC` is set?

Reply:
Ah, you are right. Didn't read the code correctly. Sorry

Reply:
No prob, thanks for the extra set of eyes.